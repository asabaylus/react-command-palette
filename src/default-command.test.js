import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import serializer from "enzyme-to-json/serializer";

import mockCommands from "./__mocks__/commands";
import DefaultCommand from "./default-command";

expect.addSnapshotSerializer(serializer);

// React 16 Enzyme adapter
Enzyme.configure({ adapter: new Adapter() });

describe("DefaultCommand", () => {
  it("should render", () => {
    const defaultCommand = shallow(
      <DefaultCommand suggestion={mockCommands[0]} />
    );
    expect(defaultCommand).toMatchSnapshot();
  });
});
