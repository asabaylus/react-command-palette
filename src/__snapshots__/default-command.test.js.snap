// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultCommand should render 1`] = `
<div
  className="item"
>
  <span>
    Start All Data Imports
  </span>
</div>
`;
